database/seeds/UserBeatmapSeeder.php

```diff
@@ -0,0 +1,350 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UserBeatmapSeeder extends Seeder
```

Comment:
You moved user generating from the model factory to a seeder, but you forgot to change the tests appropriately.

Also, wouldn't a model factory be more useful to generate a single model on demand, as in testing? @nanaya?